caravel/assets/webpack.config.js

```diff
@@ -7,6 +7,11 @@ const APP_DIR = path.resolve(__dirname, './');
 // output dir
 const BUILD_DIR = path.resolve(__dirname, './dist');
 
+const fs = require('fs');
+const contents = fs.readFileSync('package.json');
+const JSONObject = JSON.parse(contents);
```

Comment:
Somehow this part did not re-run when js file is changed, not sure when exactly does webpack reload its config.js file